src/extension.ts

```diff
@@ -354,6 +363,47 @@ export function activate(context: vscode.ExtensionContext) {
     }));
 
     //
+    // Import of a GLB file and writing out its various chunks.
+    //
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbOpen', () => {
+        const options: vscode.OpenDialogOptions = {
+             canSelectMany: false,
+             openLabel: 'Import',
+             filters: {
+                'Binary glTF': ['glb'],
+                'All files': ['*']
+            }
+        };
+
+        vscode.window.showOpenDialog(options).then(fileUri => {
+            if (fileUri && fileUri[0]) {
+                try {
+                    GlbImport.load(fileUri[0].fsPath);
```

Comment:
This should `await` the `.load` as well, in order to catch all errors.  See other notes in this review.